src/test/java/org/reaktivity/k3po/nukleus/ext/DuplexIT.java

```diff
@@ -71,6 +74,27 @@ public void shouldHandshakeWithExtension() throws Exception
 
     @Test
     @Specification({
+        "handshake.authorized/client",
+        "handshake.authorized/server"
+    })
+    public void shouldHandshakeWithAuthorization() throws Exception
+    {
+        k3po.finish();
+    }
+
+    @Test
+    @Specification({
+        "handshake.unequal.authorization/client",
+        "handshake.unequal.authorization/server"
+    })
+    public void shouldFailHandshakeWithUnequalAuthorization() throws Exception
+    {
+        thrown.expect(allOf(isA(ComparisonFailure.class), hasMessage(containsString("Handshake fail"))));
```

Comment:
Is this because we are missing a `connect aborted` feature in `k3po`?

Reply:
yes. Any other equivalent scripts for half duplex and simplex I can to connect connected write aborted. But in duplex case even if I do that the test fails with the Handshake fail exception. I actually think that is perfectly satisfactory behavior. The test also fails fast unlike some other cases where we test for exceptions (script diff exceptions based on the timeout).

Reply:
Please file a `k3po` enhancement to support `connect aborted` explicitly.

Reply:
https://github.com/k3po/k3po/issues/454

Reply:
Perfect, thanks.